staging/src/k8s.io/kubectl/pkg/cmd/diff/prune.go

```diff
@@ -52,7 +52,7 @@ func newPruner(dc dynamic.Interface, m meta.RESTMapper, r []prune.Resource) *pru
 
 func (p *pruner) pruneAll() ([]runtime.Object, error) {
 	var allPruned []runtime.Object
-	namespacedRESTMappings, nonNamespacedRESTMappings, err := prune.GetRESTMappings(p.mapper, p.resources)
+	namespacedRESTMappings, nonNamespacedRESTMappings, err := prune.GetRESTMappings(p.mapper, p.resources, false)
```

Comment:
Why is this hardcoded to false?

Reply:
`pruneAll()` will get both namespaced and nonNamespaced rest mapping. So here, namespaceSpecified should always be false here.

Reply:
Fixed.